src/test/kotlin/com/zepben/evolve/streaming/get/testdata/FeederNetworkWithCurrent.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2021 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.streaming.get.testdata
+
+import com.zepben.evolve.cim.iec61968.common.Location
+import com.zepben.evolve.cim.iec61968.common.PositionPoint
+import com.zepben.evolve.cim.iec61970.base.core.PhaseCode
+import com.zepben.evolve.cim.iec61970.base.core.PowerSystemResource
+import com.zepben.evolve.services.network.NetworkService
+import com.zepben.evolve.services.network.testdata.*
+import com.zepben.evolve.services.network.tracing.Tracing
+
+object FeederNetworkWithCurrent {
+    //                c1       c2      c3       c4
+    //    source-fcb------fsp------tx------sw--------tx2
+    //                                   (open)
+    fun create(): NetworkService {
+        val networkService = NetworkService()
+
+        val source = createSourceForConnecting(networkService, "source", 1, PhaseCode.AB)
+        val fcb = createSwitchForConnecting(networkService, "fcb", 2, PhaseCode.AB)
+        val fsp = createNodeForConnecting(networkService, "fsp", 2, PhaseCode.AB)
+        val tx = createPowerTransformerForConnecting(networkService, "tx", 2, PhaseCode.AB, 0, 0)
+        val tx2 = createPowerTransformerForConnecting(networkService, "tx2", 2, PhaseCode.AB, 0, 0)
+        val sw = createSwitchForConnecting(networkService, "sw", 2, PhaseCode.AB)
+        PhaseCode.AB.singlePhases().forEach { sw.setOpen(true, it) }
```

Comment:
You can call `setOpen` with no phase to open all phases

Reply:
silly me.... i should know this